src/radtest/radtest.c

```diff
@@ -9,22 +9,26 @@
 
 #include <stdio.h>
 #include <ado_modules.h>
+#include <mod/ado_sdcard.h>
 
 #include "../mod/ado_timers.h"
 #include "../mod/l3_sensors.h"
 #include "../mod/l2_debug_com.h"
 #include "../mod/thr/thr.h"
 #include "../mod/l7_climb_app.h"
+#include "../mod/tim/obc_time.h"
+#include "../mod/mem/obc_memory.h"
+
+#include "memtest.h"
 
-#define 			RTST_MODNR					0x55			// ('U')  Module number for rad tests only
-#define 			RTST_EVENTID_HEARTBEAT		0x55			// ('U')  Event ID for heartbeat event
 
 #define 			RTST_TICK_MS				1000			// IRQ every second.
-#define 			RTST_HEARTBEAT_TICKS		  10			// Heartbeat every 10 seconds
-#define				RTST_MEMTST_TICKS			  16			// Memorytest all 15 seconds
-#define				RTST_SENSOR_TICKS			  13			// I2C internal sensor ticks every 10 sec
+#define 			RTST_HEARTBEAT_TICKS		  60			// Heartbeat (aka Logbery watchdog) timing as in radtest1 60 sec.
+#define				RTST_MEMTST_TICKS			  1				// different tick timers are handled in memtest.c -> tick every second here.
+#define				RTST_SENSOR_TICKS			  30			// I2C internal sensor ticks every 10 sec
```

Comment:
Again !! 

Lets please confirm with Trebersburg/Carsten/Abbas   at which frequency those tests should be carried out !

Reply:
ok, for the memory:
The "write tests" are used to 'reset' and change the bit pattern stored in memory to its expected values. The read tests then can detect bit flips. As I do count the errors we can see additional bitflips after every 'read test'. So I would suggest to do the reads every 5 sec or so. The write is not so important. The only thing it does is to reestablish the pattern -> and if we keep the verbose log (showing each bit flip position (!?)) then the log gets quieter again (if there are errors detected).. So I would suggest to make this at 60 or even 120 seconds.